coderd/notifications/testdata/rendered-templates/webhook/TemplateUserAccountActivated.json.golden

```diff
@@ -0,0 +1,29 @@
+{
+  "_version": "1.1",
+  "msg_id": "00000000-0000-0000-0000-000000000000",
+  "payload": {
+    "_version": "1.1",
+    "notification_name": "User account activated",
+    "notification_template_id": "00000000-0000-0000-0000-000000000000",
```

Comment:
One more comment:

Can we keep `notification_template_id` as is, and not zero it? These values are const/stable.